Describe empty topics in DataViz and Sortable (#954)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/GridBase/1 Configuration/rowDragging/onReorder.md

#include draganddrop-ref-itemelementparam

##### field(e.promise): Promise<void>

> Row dragging is finished once this Promise is resolved.

Reviewer: ```suggestion
A Promise that is resolved when the row is dropped.
```

Author: Alyar said the following: Я бы сказал, что перетаскивание строки завершится после того как промис будет зарезолвен.
Seems like the order is the opposite to your suggestion.
